main/src/main/res/values/strings.xml

```diff
@@ -459,52 +460,52 @@
     <string name="crtext_requested">Answer challenge to continue VPN authentication</string>
-    <string name="state_auth_pending">Authentication pending</string>
+    <string name="state_auth_pending">Authentication pending…</string>
     <string name="external_authenticator">External Authenticator</string>
     <string name="configure">Configure</string>
-    <string name="extauth_not_configured">External Authenticator not configured</string>
-    <string name="faq_killswitch_title">Block non VPN connection (\"Killswitch\")</string>
-    <string name="faq_killswitch">It is often desired to block connections without VPN. Other apps often use markting terms like \"Killswitch\" or \"Seamless tunnel\" for this feature. OpenVPN and this app offer persist-tun, a feature to implement this functionality.&lt;p>The problem with all these methods offered by apps is that they can only provide best effort and are no complete solutions. On boot, app crashing and other corner cases the app cannot ensure that this block of non VPN connection works. Thus giving the user a false sense of security.&lt;p>The &lt;b>only&lt;/b> reliable way to ensure non VPN connections are blocked is to use Android 8.0 or later and use the \"block connections without VPN\" setting that can be found under Settings > Network &amp; Internet > Advanced/VPN > OpenVPN for Android > Enable Always ON VPN, Enable Block Connections without VPN</string>
+    <string name="extauth_not_configured">External two-factor app not configured</string>
+    <string name="faq_killswitch_title">Block non-VPN connections (\"Killswitch\")</string>
+    <string name="faq_killswitch">It is often desired to block connections without VPN. Other apps often use markting terms like \"Killswitch\" or \"Seamless tunnel\" for this feature. OpenVPN and this app similarly offers persistent TUN.&lt;p>The problem with all these methods offered by apps is that they can only provide best effort and are not complete solutions. Upon starting the device, app crashing and other corner cases happen because the app cannot ensure this block of non-VPN connection works. Thus the user is given a false sense of security.&lt;p>The &lt;b>only&lt;/b> reliable way to ensure non VPN connections are blocked is to use Android 8.0 or later and use the \"Block connections without VPN\" setting that can be found in Settings → Network and Internet → Advanced/VPN > OpenVPN for Android > Enable Always ON VPN, Enable Block Connections without VPN</string>
     <string name="summary_block_address_families">This option instructs Android to not allow protocols (IPv4/IPv6) if the VPN does not set any IPv4 or IPv6 addresses.</string>
     <string name="title_block_address_families">Block IPv6 (or IPv4) if not used by the VPN</string>
     <string name="install_keychain">Install new certificate</string>
-    <string name="as_servername">AS servername</string>
+    <string name="as_servername">As servername</string>
```

Comment:
"As" correct? (I don't know what "AS" is)

Reply:
Yes. AS is correct. You don't know what it is and still changed it ...

Reply:
Tried searching for it to no avail. Will add a comment.